src/main/java/ru/svgogin/service/spark/service/SparkService.java

```diff
@@ -43,14 +44,18 @@ public CompanyDto update(CompanyDto companyFromDb, CompanyDto companyDto) {
   }
 
   public CompanyDto save(CompanyDto companyDto) {
-    Company company = toEntity(companyDto);
-    if (sparkRepositoryDb.existsByInn(company.getInn())) {
-      throw new IllegalArgumentException("Company with Inn = "
-                                         + company.getInn()
-                                         + " already exists");
-    } else {
-      return toDto(aggregateTemplate.insert(company));
+    var companyInDb = sparkRepositoryDb.existsByInn(companyDto.getInn());
+    if (companyInDb) {
+      log.info("Company with inn "
```

Comment:
you should choose another log lever or it is better if you log it into exception handler